client/components/forms/form-text-input-with-action/index.jsx

```diff
@@ -0,0 +1,129 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes } from 'react';
+import classNames from 'classnames';
+import { keys, omit, noop } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import FormTextInput from 'components/forms/form-text-input';
+import FormButton from 'components/forms/form-button';
+
+export default React.createClass( {
```

Comment:
Since FB are planning to move away from .createClass in the future ([source](https://facebook.github.io/react/blog/2015/03/10/react-v0.13.html)) would it be better to use the 'extends Component' syntax here?

Also, I think the 'babel-plugin-transform-react-display-name' package infers the displayName through assignment like so:
`export default class FormTextInputWithAction extends Component( { ... } );`
I know it does for .createClass but I think it works for the es6 syntax above too.